proto/attestation/endorsement.proto

```diff
@@ -30,11 +30,15 @@ message TransparentReleaseEndorsement {
   // https://github.com/project-oak/transparent-release/blob/main/docs/claim-transparency.md#the-claim-format
   bytes endorsement = 1;
 
+  // If non-empty, then the endorsed data is not the actual binary, but this
+  // piece of data, which is usually a serialized proto containing digests.
+  bytes attachment = 2;
```

Comment:
I would not expect the endorsement to even know about attachments at all. If there is an attachment, isn't that just the subject of the endorsement? At this level, I think the optionality should be: do we want to carry the inlined version of the bytes of the subject (e.g. for attachments), or should we skip them because the recipient does not need them (e.g. for binaries, or Wasm modules, etc).